src/elements/TextInput.tsx

```diff
@@ -33,10 +33,7 @@ const StyledInputElement = styled.input`
   font-size: ${theme.inputField.fontSize};
   font-weight: ${theme.inputField.fontWeight};
   line-height: ${theme.inputField.lineHeight};
-
-  input {
-    padding: 0;
-  }
+  padding: ${theme.inputField.padding};
```

Comment:
I moved the padding to here so we can increase the clickable area for an input to be interacted with.

Reply:
Nice, definitely a good idea.